deeplake/core/dataset/deeplake_cloud_dataset.py

```diff
@@ -60,6 +60,7 @@ def token(self):
     def _set_org_and_name(self):
         if self.is_actually_cloud:
             if self.org_id is not None:
```

Comment:
@FrancescoSaverioZuppichini We can change this condition to `if self.org_id is not None and self.ds_name is not None` and I will add the required exceptions to the dataset creation step.

Reply:
done it :)